assets/src/components/incomingBox.tsx

```diff
@@ -49,10 +49,10 @@ const IncomingBoxVehicle = ({
   return (
     <VehicleTooltip vehicleOrGhost={vehicleOrGhost}>
       <button
-        className={`m-incoming-box__vehicle ${selectedClass}`}
+        className={`c-incoming-box__vehicle ${selectedClass}`}
```

Comment:
suggestion (non-blocking): Looks like an opportunity to update `selectedClass` to use BEM with `--selected` as a modifier